packages/cli/src/test/setup.js

```diff
@@ -54,3 +56,20 @@ function setArtifactDefaults() {
   const defaults = process.env.SOLIDITY_COVERAGE ? DEFAULT_COVERAGE_TX_PARAMS : DEFAULT_TESTING_TX_PARAMS;
   Contracts.setArtifactsDefaults(defaults);
 }
+
+export function stubContractUpgradeable(sandbox) {
```

Comment:
It's a minor thing, but I'd avoid exporting functions from modules that have side effects. Maybe place this funcion on a different file?

Reply:
I totally agree on not exporting function from modules that have side effects but it seems to so common in JavaScript world 💥